src/components/EmailForm/index.jsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from "react"
+import { Link } from "react-router-dom"
+
+function EmailForm(){
+    const [email,setEmail]=useState('')
+    const [subject,setSubject]=useState('')
+    const [message,setMessage]=useState('')
+    const handleSubmit=(e)=> {
```

Comment:
Add an empty line above this - and add spaces before and after the =, and one before =>